### DISCUSSION TITLE
Wrong time in backrest UI

### DISCUSSION THREAD

**Discussion by EdyTheCow** [garethgeorge/backrest#301] [Q&A] [answered]
Backups seem to be 2 hours behind the correct time. I have tried using TZ variable and tried attaching `/etc/localtime:/etc/localtime:ro` volume but it's still 2 hours behind. The logs themselves from backrest show correct time. I am not really sure what else to do to fix this?

**Comment by garethgeorge** [comment, marked as answer]
Hey, this is something that generates a bit of confusion (justifiably) so to add a bit of info that might help with debugging

 * TZ=<location> is propagated to the container but Backrest mostly uses unix epoch time. The timezone only informs the timezone used to parse cron expressions (e.g. ensure that midnight in your cron matches midnight in your local timezone).
 * Separately, the WebUI receives operations from the server with timestamps as epoch times and interprets those relative to the local timezone indicated by your browser. 

It's possible that your browser's timezone is getting interpreted incorrectly (though I'm not sure why this would be happening)?

**Reply by EdyTheCow** [reply]
Ah, I see. You were on point, this was browser related issue. I am running a hardened firefox which shares a very limited amount of info with websites. Testing it on a different browser worked perfectly fine, should of thought to check this case before posting here. Thank you for a quick reply!

**Reply by dadav** [reply]
Had a similar problem. Disabling `privacy.resistFingerprinting` fixed it for me (firefox setting).
